pyproject.toml

```diff
@@ -117,6 +117,7 @@ dependencies = [
   "xhtml2pdf==0.2.15",
   "python-bidi==0.4.2",
   "python-dateutil==2.9.0",
```

Comment:
Don't we need to install `sqlite` here

Reply:
I'll take care of this in the follow up PR.